### DISCUSSION TITLE
webBlob.readable stream is not getting intercepted properly with Remix 1.5.1 and MWS-0.42.1

### DISCUSSION THREAD

**Discussion by spencersmb** [mswjs/msw#1285] [Q&A]
I followed Kent C. Dodds testing tutorials and had MWS working with Remix 1.4.3 just fine with Jest v28. I updated Remix to 1.5.1 and it now uses webBlob.readable stream with Fetch calls, however MWS doesn't seem to support this. Has anyone else experienced this yet?

**Comment by kettanaito** [comment]
Hey, @spencersmb. Thanks for reporting this!

Please, do you have a reproduction repository where I could look into this?

**Reply by spencersmb** [reply]
@kettanaito Hi,
I'm digging into a Remix issue I'm also having without the use of MSW I encountered while setting up a repo to post. Maybe it will all get fixed once I get the other issue solved. Either way I'll circle back here with a response. 

Thanks for the quick response!
